### PR TITLE
core: cleanup unused imports

### PR DESCRIPTION
**Pull request by nitikagarw**
References #xxxx

### CODE REVIEW

File: akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala [hunk 1]
```diff
@@ -4,8 +4,6 @@
 
 package akka.http.impl.engine.http2
 
-import akka.http.ccompat._
```
Comment (jrudolph) [inline akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala:7]: All the ccompat ones are actually needed but only for Scala 2.12.

Comment (Marcus-Rosti) [reply akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala:7]: I think cats and any other library that uses implicits heavily has a lot of issues with ignoring unused imports.

**Review by jrudolph — APPROVED** [review]
LGTM (I removed the changes to files that shouldn't have been changed)

**Comment by nitikagarw** [comment]
Thanks, @jrudolph. The aim of this PR was to mostly reduce the compile warnings and have cleaner logs on the terminal.

**Comment by jrudolph** [comment]
> The aim of this PR was to mostly reduce the compile warnings and have cleaner logs on the terminal.

Yes, unfortunately, the `ccompat` imports are needed for 2.12 builds, so we need a different mechanism to remove the warning.

**Pull request merged**